.gitignore

```diff
@@ -15,6 +15,8 @@
 !*/src/*
 !*/include/*
 !*/tests/*
+!cmake/git_watcher.cmake
```

Comment:
Could you please remove these. They are needed in case we change the files.

Reply:
> Could you please remove these. They are needed in case we change the files.

I added these lines because git wasn't tracking the files otherwise. I guess another line made git ignore them so I forced git to **not** ignore them using a "!" in front of their paths.
So I guess these lines should stay right ? Or maybe the lines that made git ignore them should be tweaked ?

Reply:
You could do `git add` those specific files without putting it in gitignore. I can try and fix the clang-format error for you